plugins/versionpress/setup-hooks.php

```diff
@@ -78,8 +78,7 @@
 //----------------------------------
 
 add_filter('automatic_updates_is_vcs_checkout', function () {
-    $forceUpdate = UninstallationUtil::uninstallationShouldRemoveGitRepo(); // first commit was created by VersionPress
-    return !$forceUpdate; // 'false' forces the update
+    return !VersionPress::isActive(); // 'false' forces the update
```

Comment:
I think we should use the word "allow" or "enable" instead of "force", what do you think?

```suggestion
    return !VersionPress::isActive(); // 'false' enables auto-updates
```

Reply:
That's fine with me - I used the word "force" as that's what the original comment said :-)

Reply:
Yep, we didn't choose the best word. If you agree, please commit the suggestion and with that, the PR will be ready to be merged.

Reply:
Done!